client/src/Firebase/firebase.js

```diff
@@ -0,0 +1,36 @@
+import app from 'firebase/app';
+import 'firebase/auth';
+
+const config = {
+    apiKey: process.env.REACT_APP_API_KEY,
+    authDomain: process.env.REACT_APP_AUTH_DOMAIN,
+    databaseURL: process.env.REACT_APP_DATABASE_URL,
+    projectId: process.env.REACT_APP_PROJECT_ID,
+    storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
+    messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
+};
+
+class Firebase {
+    constructor() {
+        app.initializeApp(config);
+
+        this.auth = app.auth();
+    }
+
+    // *** Auth API ***
+
+    doCreateUserWithEmailAndPassword = (email, password) =>
```

Comment:
This is great, looks very intuitive. Curious to see how this looks like in the database though.